slacker.py

```diff
@@ -20,20 +26,53 @@ def __init__(self):
         self.logger.setLevel(logging.INFO)
         self.logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
I usually use the next configuration for logging:
```python
self.logger = logging.getLogger("DBEngine")
self.logger.setLevel(logging.INFO)

handler = logging.StreamHandler(sys.stdout)
handler.setLevel(logging.DEBUG)
formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s', 
                              datefmt='%m.%d.%Y-%I:%M:%S')
handler.setFormatter(formatter)
self.logger.addHandler(handler)
```

I think it's important to know the log level, where it was happened and time of this event.

Reply:
fixed